### PR TITLE
Bluesky documentation API update

### PR DESCRIPTION
**Pull request by jrmlhermitte**
@danielballan 
from https://github.com/NSLS-II/NSLS-II.github.io/pull/7

Update of some of the bluesky documentation using the new API.

### CODE REVIEW

File: source/_cookbook/simulation-mode.py [hunk 1]
```diff
@@ -38,21 +38,22 @@
 
 print_summary(count([det]))
 print_summary(scan([det], motor, 1, 3, 3))
-print_summary(outer_product_scan([det], motor1, 1, 3, 3, motor2, 1, 2, 2, False))
+print_summary(outer_product_scan([det], motor1, 1, 3, 3, motor2, 1, 2, 2,
```
Comment (tacaswell) [inline source/_cookbook/simulation-mode.py]: This should be `grid` now I think.

Comment (danielballan) [reply source/_cookbook/simulation-mode.py]: Yeah, and now `print_summary` warns: it should be `summarize_plan`. (Sorry for the annoying name changes! Now that we have 1.0 I promise they will diminish.)

Comment (jrmlhermitte) [reply source/_cookbook/simulation-mode.py]: @tacaswell yes you're right thanks for the information!
(https://github.com/NSLS-II/bluesky/blob/master/bluesky/plans.py#L1420)
@danielballan thanks for this information as well, tested on my machine and updated.
(will wait for Travis error again, I wasn't too sure what it was about)

**Comment by jrmlhermitte** [comment]
I'm not sure why errors fail. Locally, failure happens on `from ophyd.sim import motor, det`.
However, in my same environment, this command works.
Log:
<details>
<summary>TypeError: expected string or bytes-like object</summary>

```

WARNING: /home/julienl/research/projects/nsls2-docs/source/_cookbook/adaptive-steps.py failed to execute correctly: Traceback (most recent call last):
  File "/home/julienl/research/projects/nsls2-docs/source/_cookbook/adaptive-steps.py", line 31, in <module>
    from ophyd.sim import motor, det
  File "/home/julienl/research/projects/ophyd/ophyd/sim.py", line 866, in <module>
    globals().update(hw().__dict__)
  File "/home/julienl/research/projects/ophyd/ophyd/sim.py", line 780, in hw
    motor = SynAxis(name='motor')
  File "/home/julienl/research/projects/ophyd/ophyd/sim.py", line 269, in __init__
    super().__init__(name=name, parent=parent)
  File "/home/julienl/research/projects/ophyd/ophyd/device.py", line 739, in __init__
    [getattr(self, attr) for attr, cpt in self._sig_attrs.items()
  File "/home/julienl/research/projects/ophyd/ophyd/device.py", line 740, in <listcomp>
    if not cpt.lazy]
  File "/home/julienl/research/projects/ophyd/ophyd/device.py", line 176, in __get__
    instance._signals[self.attr] = self.create_component(instance)
  File "/home/julienl/research/projects/ophyd/ophyd/device.py", line 131, in create_component
    cpt_inst = self.cls(parent=instance, **kwargs)
  File "/home/julienl/research/projects/ophyd/ophyd/signal.py", line 56, in __init__
    from . import control_layer as cl
  File "/home/julienl/research/projects/ophyd/ophyd/control_layer.py", line 13, in <module>
    from ._pyepics_shim import setup, caput, caget, get_pv, pv_form, thread_class
  File "/home/julienl/research/projects/ophyd/ophyd/_pyepics_shim.py", line 223, in <module>
    pv_form = get_pv_form(epics.__version__)
  File "/home/julienl/research/projects/ophyd/ophyd/_pyepics_shim.py", line 207, in get_pv_form
    version = parse_version(_fix_git_versioning(version))
  File "/home/julienl/anaconda3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 211, in parse_version
    return SetuptoolsVersion(v)
  File "/home/julienl/anaconda3/lib/python3.6/site-packages/pkg_resources/_vendor/packaging/version.py", line 200, in __init__
    match = self._regex.search(version)
TypeError: expected string or bytes-like object

```
</details>

**Comment by danielballan** [comment]
Hmm. 10 minutes of digging did not make the problem clear. I'll try to dig deeper shortly.

**Comment by jrmlhermitte** [comment]
same (hence the big delay here :-/) will also try to dig in later

**Comment by tacaswell** [comment]
what version of pyepics is being used?

**Pull request closed without merging**